pkg/controller/decomission.go

```diff
@@ -30,6 +30,25 @@ import (
 // checkForPoolDecommission validates the spec of the tenant and it's status to detect a pool being removed
 func (c *Controller) checkForPoolDecommission(ctx context.Context, key string, tenant *miniov2.Tenant, tenantConfiguration map[string][]byte) (*miniov2.Tenant, error) {
 	var err error
+	// delete useless status.pools first
```

Comment:
I think is actually code to remove duplicate pools in the status field right? can we rename the variables to reflect this? I think it would make the intention of the code block more useful